tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/CreatedumpTests.cs

```diff
@@ -371,6 +371,28 @@ public async Task NativeCrash()
         File.Exists(reportFile.Path).Should().BeTrue();
     }
 
+    [SkippableFact]
+    public async Task ResumeProcessWhenCrashing()
+    {
+        SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        SkipOn.PlatformAndArchitecture(SkipOn.PlatformValue.Windows, SkipOn.ArchitectureValue.X86);
+
+        using var reportFile = new TemporaryFile();
+
+        using var helper = await StartConsoleWithArgs(
+                               "crash-native",
+                               enableProfiler: true,
+                               [LdPreloadConfig, CrashReportConfig(reportFile), ("DD_INTERNAL_CRASHTRACKING_CRASH", "1")]);
+
+        var completion = await Task.WhenAny(helper.Task, Task.Delay(TimeSpan.FromMinutes(1)));
```

Comment:
One minute _seems_ like a long time, but, CI... we use 10 minutes elsewhere. Just sayin...